### ISSUE TITLE
MP3 files

### ISSUE THREAD

**Issue by KrzysioMisio** [WWBN/AVideo#7747]
Hello Daniel
there is a problem with uploading MP3 files
on the public encoder it does not work either
there is also a problem on the demo site

**Comment by DanielnetoDotCom** [comment]
describe exactly what you are doing in a way I can reproduce (step by step). otherwise, I cannot help you

**Comment by KrzysioMisio** [comment]
I select the AUDIO button in the encoder
I select an MP3 file from my computer's hard drive and the file loads,
After loading the file, the encoder does not process the file and stops. 
It doesn't even show the progress bar
in "my Video" the file has EE EE EE

**Video files load normally !!**

**Comment by DanielnetoDotCom** [comment]
What happened, is it fixed?

**Comment by KrzysioMisio** [comment]
no, for many months I used Shotcut to edit mp3 and mp4 files and uploading such files always worked
Now after AVideo updates, the files no longer work
I don't understand why MP4 works and MP3 doesn't
I found another Movavi Video Editor Plus program for editing and I can upload to AVideo again

**Comment by DanielnetoDotCom** [comment]
Just tested on demo site and works fine

**Comment by KrzysioMisio** [comment]
Daniel install Ubuntu 22.04 and php8.2 for yourself
because it seems to me that you are using ubuntu 16 and php7.2 on the demo site, that's why you don't see problems and errors in others
Today I had a problem with uploading a mp4 movie that I had already uploaded before.
The progress bar did not display and in My Videos the video had EE EE EE EE
You can see that this is not only a problem with MP3 files

`[07-Mar-2023 10:58:40 Europe/Warsaw] PHP Warning:  Undefined variable $stmt in path/to/my/streamer/site/objects/mysql_dal.php on line 219`

`[07-Mar-2023 10:58:40 Europe/Warsaw] AVideoLog::ERROR: [sqlDAL::readSql] (mysqlnd) Prepare failed again return false SCRIPT_NAME: /objects/aVideoEncoderNotifyIsDone.json.php`

**Comment by DanielnetoDotCom** [comment]
The error above is not enough, you need to provide more from it. 

In my local environment I have Ubuntu 22 and PHP 8.1.6

**Comment by KrzysioMisio** [comment]
ok, I looked at the demo page
I don't have more errors, except Scheduler
